### DISCUSSION TITLE
[insight]: Calculate panel data based on the local time zone

### DISCUSSION THREAD

**Discussion by zhujiageng** [directus/directus#19378] [Feature Requests]
### Summary

calculate panel data based on the local time zone, instead of based on the UTC standard time zone

### Motivation

Our suppliers produce daily reports based on the local time zone, while Directus generates daily reports in UTC. As a result, the data from both sides don't match. My local time zone is UTC+8, so the data from Directus is from 8 AM local time to 8 AM the next day, not from midnight to midnight local time.

### Detailed Design

Add a switch in the Panel settings, which, when turned on, uses the local time zone for data calculations; otherwise, calculations are based on the UTC standard time zone.

### Must Have:
- a switch in the Panel settings to enable calculations based on local timezone

### Drawbacks

- this is not a break change

### Alternatives

none

### Adoption Strategy

This switch can be turned off by default, so it will have no impact on existing users.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
